client/dist/styles.css

```diff
@@ -39,6 +38,12 @@
   font-style: normal;
   font-weight: 900;
   src: local('Airbnb Cereal App Black'), url('./fonts/airbnb-cereal-app-cufonfonts-woff/AirbnbCerealBlack.woff') format('woff');
+} */
+
+/*set images to responsive*/
+img {
+  max-width: 100%;
```

Comment:
Suggestion: I don't think we need max-width here, I think regular width would be just fine. Up to you.